### PR TITLE
make title scrolling consistent in extras slider

### PR DESCRIPTION
**Pull request by sevenrats**
only scroll name labels in the extras slider if they have focus.  this is more consistent and more visually appropriate.

**Changes**
only scroll pLabel in ExtrasItem if it has focus.

### CODE REVIEW

File: components/extras/ExtrasItem.xml [hunk 1]
```diff
@@ -8,8 +8,11 @@
   <children>
     <LayoutGroup layoutDirection="vert" >
       <Poster id="posterImg" width="234" height="300" translation="[8,243]" failedBitmapUri="pkg:/images/baseline_person_white_48dp.png" />
-      <ScrollingLabel id="pLabel" horizAlign="center" vertAlign="bottom" maxWidth="266" font="font:SmallestBoldSystemFont" height="48"  />
+      <ScrollingLabel id="pLabel"
+        horizAlign="center" vertAlign="bottom"
+        maxWidth="266" height="48"
+        font="font:SmallestBoldSystemFont" repeatCount="0"/>
       <Label id="SubTitle" horizAlign="center" vertAlign="center" font="font:SmallestBoldSystemFont" height="32" color="#A7A7A7FF"  />
```
Comment (candry7731) [inline components/extras/ExtrasItem.xml]: should make this a scrolling label also as this is the characters title and is almost always cutoff 
```suggestion
            <ScrollingLabel id="SubTitle" horizAlign="center" vertAlign="center" font="font:SmallestBoldSystemFont" maxWidth="246" height="32" color="#A7A7A7FF" repeatCount="0"  />
  />
```

File: components/extras/ExtrasItem.brs [hunk 2]
```diff
@@ -22,6 +22,12 @@ sub showContent()
 end sub
 
 sub focusChanged()
+    if m.top.itemHasFocus = true
+        m.name.repeatCount = -1
```
Comment (candry7731) [inline components/extras/ExtrasItem.brs:26]: need to make the scroll work for the role text
```suggestion
        m.name.repeatCount = -1
        m.role.repeatCount = -1
```

File: components/extras/ExtrasItem.brs [hunk 3]
```diff
@@ -22,6 +22,12 @@ sub showContent()
...
+    else
+        m.name.repeatCount = 0
```
Comment (candry7731) [inline components/extras/ExtrasItem.brs:29]: same

```suggestion
        m.name.repeatCount = 0
        m.role.repeatCount = 0

```

File: components/extras/ExtrasItem.xml [hunk 4]
```diff
@@ -8,8 +8,14 @@
...
     <LayoutGroup layoutDirection="vert" >
       <Poster id="posterImg" width="234" height="300" translation="[8,243]" failedBitmapUri="pkg:/images/baseline_person_white_48dp.png" />
-      <ScrollingLabel id="pLabel" horizAlign="center" vertAlign="bottom" maxWidth="266" font="font:SmallestBoldSystemFont" height="48"  />
-      <Label id="SubTitle" horizAlign="center" vertAlign="center" font="font:SmallestBoldSystemFont" height="32" color="#A7A7A7FF"  />
-    </LayoutGroup>
+      <ScrollingLabel id="pLabel"
+        horizAlign="center" vertAlign="bottom"
+        maxWidth="266" height="48"
+        font="font:SmallestBoldSystemFont" repeatCount="0"/>
+      <ScrollingLabel id="SubTitle"
```
Comment (1hitsong) [inline components/extras/ExtrasItem.xml:15]: The change of this from a `label` to a `scrollinglabel` causes this warning to be logged.

```
=================================================================
Warning occurred while setting a field of an RoSGNode
-- Tried to set nonexistent field "width" of a "ScrollingLabel" node
pkg:/components/extras/ExtrasItem.brs(14)
=================================================================
```

**Review by 1hitsong — CHANGES_REQUESTED** [review]

**Review by 1hitsong — APPROVED** [review]
To be merged 2/12 unless an issue arises.

**Pull request merged**